ethereum/spec/src/main/java/tech/pegasys/teku/spec/Spec.java

```diff
@@ -1095,4 +1108,22 @@ public boolean equals(final Object o) {
   public int hashCode() {
     return Objects.hash(forkSchedule);
   }
+
+  public Optional<Integer> getMaxBlobsPerBlockAtSlot(final UInt64 slot) {
```

Comment:
It's used `.orElseThrow()` in all calls of this method. Does it make sense? Maybe return Integer and default -> throw?

Reply:
orElseThrow because its a bug if you get there, have a deneb/electra/fulu milestone and dont have these things